examples/codepush/app/lib/bugsnag.js

```diff
@@ -3,7 +3,10 @@
 // anywhere.
 //-------------------------------------------------------------------------------------------------
 // https://docs.bugsnag.com/platforms/react-native/#basic-configuration
-import { Client } from 'bugsnag-react-native';
-const client = new Client();
+import { Client, Configuration } from 'bugsnag-react-native';
+
+const config = new Configuration('YOUR_API_KEY_HERE');
+config.codeBundleId = '1.0.0-b12'
```

Comment:
is setting this important to the example? If so we should document why this is being done

Reply:
Yes - I've added a link to the relevant docs.